nmigen_soc/wishbone/bus.py

```diff
@@ -166,6 +167,147 @@ def memory_map(self, memory_map):
         self._map = memory_map
 
 
+class Connector(Elaboratable):
```

Comment:
`Connector` seems too generic for a width up-converter. How about `Upconverter` ?

Reply:
As discussed in #18 I did use Connector because the functionality is broader then upconverting, it's functionality is to connect a single initiator to a single subordinate. Upconverting will be done when necessary.
You will need to agree with @whitequark what to go for.

Reply:
Could the `Connector` class be composed internally of two separate up- and down-converter classes ?

Reply:
I can see during implementation of the feature if it makes sense to do that but current feeling is that it will cause unnecessary code duplication with negligible improvement in code clarity or maintainability.